### PR TITLE
products layout

### PR DESCRIPTION
**Pull request by AlonaLook**

### CODE REVIEW

File: src/css/modules/cards-connecting.scss [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+
+      &--seasons {
+        background-image: url($img-path + "products/ic-seasons.svg");
+      }
+      &--status {
+        background-image: url($img-path + "products/ic-inventory-status.svg");
+      }
+    }
+
+
```
Comment (PritulaRoman) [inline src/css/modules/cards-connecting.scss]: Please, remove the white space

File: src/js/cards.js [hunk 2]
```diff
@@ -66,6 +69,24 @@ export const buildCards = () => {
       $boxFooterCards.html(footerCards);
       return response;
     })
+    .then((response) => {
+      const footerCardsInformation = response.data.results;
```
Comment (PritulaRoman) [inline src/js/cards.js]: Duplicated code `response.data.results`

File: .gitignore [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /.idea
-/dist/*
+/dist/*.js
```
[the remark this answers is not in the document]
Comment (AlonaLook) [reply .gitignore]: Changed

File: src/css/modules/cards-connecting.scss (hunk 1 above)
Comment (AlonaLook) [reply src/css/modules/cards-connecting.scss]: Removed.

**Pull request merged**